bin/node/src/commands/bundled.rs

```diff
@@ -219,9 +230,13 @@ impl BundledCommand {
         if should_start_ntb {
             let id = join_set
                 .spawn(async move {
+                    // SAFETY: The block_producer_address is always valid as it is created from a
```

Comment:
Since we're using `anyhow` anyways, it might be shorter to use `Url::parse(..).context("foo")?`.